app/javascript/components/form/components/styles.css

```diff
@@ -131,3 +131,8 @@
   right: 9px !important;
   left: initial !important;
 }
+
+.To{
+  margin-left: 307px;
+    margin-top: -103px;
```

Comment:
Correct indents, and rename this css class to something a little more verbose if still used. If not used delete it.